EESSI-remove-software.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+#
+# Script to install EESSI software stack (version set through init/eessi_defaults)
+
+# see example parsing of command line arguments at
+#   https://wiki.bash-hackers.org/scripting/posparams#using_a_while_loop
+#   https://stackoverflow.com/questions/192249/how-do-i-parse-command-line-arguments-in-bash
+
+display_help() {
+  echo "usage: $0 [OPTIONS]"
+  echo "  --build-logs-dir       -  location to copy EasyBuild logs to for failed builds"
+  echo "  -g | --generic         -  instructs script to build for generic architecture target"
```

Comment:
```suggestion

Reply:
Don't we need this to remove stuff from the generic tree?